src/generic/FreeAssAlgebra.jl

```diff
@@ -4,46 +4,50 @@
 #
 ###############################################################################
 
-export exponent_word, set_exponent_word
+export exponent_word, set_exponent_word, calc_suffix_match_vector
 
 ###############################################################################
 #
 #   Data type and parent object methods
 #
 ###############################################################################
 
-function parent_type(::Type{FreeAssAlgElem{T}}) where T <: RingElement
-   return FreeAssAlgebra{T}
+function parent_type(::Type{FreeAssAlgElem{T}}) where {T<:RingElement}
```

Comment:
It would be a bit easier to review the changes in this file if it didn't also include code formatting changes and addition of braces :-/